docs/content/integrations/airlift/tutorial/overview.mdx

```diff
@@ -0,0 +1,34 @@
+# Airflow Migration Tutorial
+
+This tutorial demonstrates using `dagster-airlift` to migrate an Airflow DAG to Dagster.
+
+Using `dagster-airlift` we can
+
+- Observe Airflow DAGs and their execution history with no changes to Airflow code
+- Model and observe assets orchestrated by Airflow with no changes to Airflow code
+- Enable a migration process that
+  - Can be done task-by-task in any order with minimal coordination
+  - Has task-by-task rollback to reduce risk
+  - That retains Airflow DAG structure and execution history during the migration
+
+## Process
+
+This is a high level overview of the steps to migrate an Airflow DAG to Dagster:
+
+- **Peer**
+  - Observe an Airflow instance from within a Dagster Deployment via the Airflow REST API.
+  - This loads every Airflow DAG as an asset definition and creates a sensor that polls Airflow for execution history.
```

Comment:
Random thought, but is there a way to filter what gets pulled in?

If not, that might end up being a need of some users.

Reply:
yes - see `dag_selector_fn` argument